openupgrade_scripts/scripts/mail/15.0.1.5/post-migration.py

```diff
@@ -104,7 +104,9 @@ def _migrate_placeholder_html(string):
     """
     Replace dynamic placeholders in HTML fields:
     Example: 'Your name is ${object.name}' -> 'Your name is <t t-out="object.name"></t>'
+    Example:'${object.html_content | safe}' -> '<t t-out="object.html_content"></t>'
     """
+    string = re.sub(r"\s?\|\s?safe\s?", "", string)
```

Comment:
This is not filtering `'${object.html_content|    safe}'` for example.

Reply:
Try now :)